### DISCUSSION TITLE
Is asyncpg support possible?

### DISCUSSION THREAD

**Discussion by omid9h** [ahopkins/mayim#47] [Q&A]
I have a usecase to insert high volume of data at peak time in Postgres and based on my benchmark `asyncpg` has at least 50% more performance. since `asyncpg` does not follow Python's DB-API, is it feasible to add support for it in mayim?

**Comment by ahopkins** [comment]
Yes, it is possible. I've been trying to think of the nicest way to handle various drivers. For example, how to choose when multiple drivers are installed?
